crates/rnote-ui/src/dialogs/import.rs

```diff
@@ -296,11 +347,13 @@ pub(crate) async fn dialog_import_xopp_w_prefs(
     let builder = Builder::from_resource(
         (String::from(config::APP_IDPATH) + "ui/dialogs/import.ui").as_str(),
     );
-    let dialog: Dialog = builder.object("dialog_import_xopp_w_prefs").unwrap();
+    let dialog: adw::Dialog = builder.object("dialog_import_xopp_w_prefs").unwrap();
     let dpi_row: adw::SpinRow = builder.object("xopp_import_dpi_row").unwrap();
     let xopp_import_prefs = canvas.engine_ref().import_prefs.xopp_import_prefs;
+    let import_xopp_button_cancel: Button = builder.object("import_xopp_button_cancel").unwrap();
+    let import_xopp_button_confirm: Button = builder.object("import_xopp_button_confirm").unwrap();
 
-    dialog.set_transient_for(Some(appwindow));
+    dialog.present(appwindow);
```

Comment:
Move this down to right before awaiting the channel receiver, so the handlers are already set up before the dialog is presented.